### ISSUE TITLE
A11y_Code.visualstudio.com_visual studio Insides page_Download visual studio insiders_High Contrast: In high contrast black and white mode the "Download for windows" link is not adopting the respective colors.

### ISSUE THREAD

**Issue by Samhitha66** [microsoft/vscode-docs#5155]
## GitHub Tags:
#A11yTCS;#Win10;#WCAG4.3.1;#DesktopWeb;#A11ySev2;#ChromiumEdge;#P1-VisualStudioCodeServices-Web-Feb2022;#P1_Visual Studio Code Services;#Visual Studio Code Services;

## Environmental Details:
#URL: https://code.visualstudio.com/insiders

## OS & Browser Details :
Microsoft Windows 10 Enterprise
Version 10.0.19044 Build 19044
Microsoft Edge
Version 99.0.1150.7 (Official build) dev (64-bit)

## Pre-Requisite:
To apply high contrast follow the below steps:

1. Open settings, tab till Ease of access and press enter.
2. Tab till High contrast press enter.
3.  From the drop down select a proper theme by pressing enter. Use all the 2 themes i.e. 'black' , 'white' one by one.
4. Tab till apply button and press enter

## Repro Steps :

1. Launch the URL https://code.visualstudio.com/insiders
2. Tab till "Download for windows" link.
3. Observe the "Download for windows" link is adopting high contrast black and white mode.

## Actual Result :
In high contrast black and white mode the "Download for windows" link is not adopting the respective colors.

## Expected Result:
In high contrast black and white mode the "Download for windows" link should  adopt the respective colors like yellow and blue.

## User Impact :
Low vision users won't be able to see the elements properly as there are not adopting the respective colors in high contrast mode.

## Attachments : 

https://user-images.githubusercontent.com/99243079/153137517-a39f2a91-e3d4-4e66-8bc5-294175af9d62.mp4

**Comment by rzhao271** [comment]
I see what's happening. We're losing the outline when in high-contrast mode, so the user just sees bold text. Because the bold text is the same colour as the non-link text, the download button looks more like a heading than an actual button.

**Comment by rzhao271** [comment]
I don't think we can change the text colour of the button, but we can add a border around it specifically when the user is in high-contrast mode.
![A screenshot showing the new download button with the border around it](https://user-images.githubusercontent.com/7199958/153312995-210d2ed1-b838-4e8a-9f26-00c0fd7c8926.PNG)

**Comment by Samhitha66** [comment]
@rzhao271 

Checked again this bug in the below environment
URL: https://code.visualstudio.com/insiders/

As per your reply, we are getting border around it specifically when the user is in high-contrast mode.
So, the issue is fixed.
![Bug_5155](https://user-images.githubusercontent.com/99243079/157196397-49615388-c09e-40de-986f-e02d6e5c491c.png)
